lib/sectioned-multi-select.js

```diff
@@ -1211,8 +1211,8 @@ class SectionedMultiSelect extends PureComponent {
                           />
                         </View>
                       ) : searchTerm ? (
-                        callIfFunction(noResultsComponent(searchTerm))
-                      ) : null}
+                        callIfFunction(renderNoResults(searchTerm)
```

Comment:
`callIfFunction` expects a function. 
I think this can just be `renderNoResults ? renderNoResults(searchTerm) : null`